recipes/icqsol/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: icqsol
+  version: "0.3.20"
+
+about:
+  home: 'https://github.com/pletzer/icqsol'
+  summary: 'A collection of utilities for constructing complex geometries from primitive shapes'
+  license: MIT
+
+source:
+  git_url: https://github.com/pletzer/icqsol.git
+  git_rev: 05b229c
+
+build:
+  number: 0
+
+test:
+  # Python imports
+  imports:
+    - csg
+    - icqsol
+    - numpy
+    - triangle
+    - vtk
+
+requirements:
+  build:
+    - cmake
+    - numpy
+    - python
+    - pycsg ==0.3.12
```

Comment:
In these specific cases where I've set versions, yes because even small changes to these underlying packages most often produce different results for the Galaxy CSG tools that depend on them.  So new versions of these underlying packages will require new Galaxy tool versions.  This is not the case for numpy or python.